sdk/extension-wcf/ci.yml

```diff
@@ -19,19 +19,16 @@ pr:
     - release/*
   paths:
     include:
-    - sdk/extensions/wcf
+    - sdk/extension-wcf
 
 extends:
   template: /eng/pipelines/templates/stages/archetype-sdk-client.yml
   parameters:
-    SDKType: wcf
-    ServiceDirectory: extensions/wcf
+    ServiceDirectory: extension-wcf
     ArtifactName: packages
     Artifacts:
     - name: Microsoft.CoreWCF.Azure.StorageQueues
       safeName: MicrosoftCoreWCFAzureStorageQueues
     - name: Microsoft.WCF.Azure.StorageQueues
       safeName: MicrosoftWCFAzureStorageQueues
-    CheckAOTCompat: false
-    TestSetupSteps:
```

Comment:
Do you still need these steps for the ci pipeline?

Reply:
Yes we need that step, i have reverted these changes.